src/Lesson6/HomeTask1.java

```diff
@@ -0,0 +1,41 @@
+package Lesson6;
+
+
+
+class A{     //Створюєм класс.
```

Comment:
Re-read a task and make changes accordingly. Don't forget about naming convention.